sre-recipes/recipes/recipe_1/recipe_1.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -*- coding: utf-8 -*-
+"""
+This module contains the implementation of recipe 1
+"""
+
+import os
+import subprocess
+from recipe import Recipe
+
+
+class Recipe1(Recipe):
+    """
+    This class implements recipe 1, which purposefully
+    introduces latency into the frontend service.
+    """
+
+    def __init__(self):
+        self.__service = 'frontend'
+
+    @staticmethod
+    def get_relative_path(relative_path, filename, extension):
+        """
+        Returns the absolute path to the filename given the relative
```

Comment:
It's kind of confusing to call it "get_relative_path" when it's returning the absolute path.

Reply:
I think this will no longer be an issue because we aren't going to use different images to switch between working and broken versions anymore, which means we won't need to be searching for different images + using these methods to find paths!